src/sources/opentelemetry/mod.rs

```diff
@@ -29,6 +36,22 @@ pub const LOGS: &str = "logs";
 #[derive(Clone, Debug)]
 #[serde(deny_unknown_fields)]
 pub struct OpentelemetryConfig {
```

Comment:
Does the cue doc need to be updated?

Reply:
Oh, I think this is going to break CUE with how we inject the TLS options. I did per server TLS options to match the opentelemetry-collector, but we don't _have_ to.

Any idea how much work it would be to get CUE to support this?

Reply:
Compromised with the expected sunsetting of our hand written CUE by manually including the tls options in the config object.